packages/react-tree-grid/package.json

```diff
@@ -11,6 +11,7 @@
   },
   "dependencies": {
     "@fluentui/react-context-selector": ">=9.1.47 < 10.0.0",
+    "@fluentui/react-jsx-runtime": ">=9.0.29 < 10.0.0",
```

Comment:
wondering why are we using these kind of semver ranges when it can be simplified to `^9.0.29` ?

Reply:
I have the faintest idea. I'm just following the pattern I've seen in the other packages, @ling1726 might know more?!

Reply:
They are equivalent. Personally I just got into the habit of using that syntax to declare peer dep. Since generally, peer deps go across several majors.

Reply:
yeah for peerDeps its valid, but for unlikely for deps